fundingmanager_test.go

```diff
@@ -687,16 +712,32 @@ func waitForOpenUpdate(t *testing.T, updateChan chan *lnrpc.OpenStatusUpdate) {
 
 func assertNoChannelState(t *testing.T, alice, bob *testNode,
 	fundingOutPoint *wire.OutPoint) {
-	state, _, err := alice.fundingMgr.getChannelOpeningState(fundingOutPoint)
-	if err != ErrChannelNotFound {
-		t.Fatalf("expected to not find channel state, but got: %v", state)
-	}
 
-	state, _, err = bob.fundingMgr.getChannelOpeningState(fundingOutPoint)
-	if err != ErrChannelNotFound {
-		t.Fatalf("expected to not find channel state, but got: %v", state)
+	assertErrChannelNotFound(t, alice, fundingOutPoint)
+	assertErrChannelNotFound(t, bob, fundingOutPoint)
+}
+
+func assertErrChannelNotFound(t *testing.T, node *testNode,
+	fundingOutPoint *wire.OutPoint) {
+
+	var state channelOpeningState
+	var err error
+	for i := 0; i < 10; i++ {
```

Comment:
Would it be possible to break out the default number of iterations and sleep duration as constants?